src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -274,7 +274,7 @@ export default class SourceFileLinter {
 			if (initializerType.flags & ts.TypeFlags.StringLiteral) {
 				let rendererName;
 				if (
-					ts.isStringLiteral(rendererMember.initializer) ||
+					ts.isStringLiteralLike(rendererMember.initializer) ||
 					ts.isNoSubstitutionTemplateLiteral(rendererMember.initializer)
```

Comment:
This check is obsolete now as `StringLiteralLike` includes `NoSubstitutionTemplateLiteral`.

Reply:
Thanks!